build/constants.yaml

```diff
@@ -27,14 +27,14 @@
     python-gunicorn-custom-thread-num: PYTHON_GUNICORN_CUSTOM_THREAD_NUM
     nginx-conf-file: NGINX_CONF_FILE
   outputs:
-    - type: csharp
```

Comment:
The spacing change is expected, it wasn't updating before but now it should after this change: https://github.com/microsoft/Oryx/pull/2213/commits/eba343091a36631513615ec7ca8518fbf67e6d96#diff-3ed663dcfb84882e18b67fce2ab60dcb67f0b57ac69932ca286dcea41b5d80a5R137